docker/utils/utils.py

```diff
@@ -715,7 +715,7 @@ def create_container_config(
         'Hostname': hostname,
         'Domainname': domainname,
         'ExposedPorts': ports,
-        'User': user,
+        'User': '{0}'.format(user) if user else None,
```

Comment:
Fair enough, I didn't want to use `str()` and didn't bother to look what was available in `six` :)